package.json

```diff
@@ -27,6 +27,7 @@
     "cardinal": "~0.5.0",
     "esprima": "~1.1.1",
     "estraverse": "~4.0.0",
+    "lodash.defaults": "^3.1.1",
```

Comment:
Just use xtend please. lodash.defaults depends on a [bunch of other things](https://www.npmjs.com/package/lodash.defaults).
Also please use `~` for ranges.

As a side note lodash on npm is a joke. Packages that should be minimal pull down half the world since the dependencies are purely done, mainly due to lodash actually being a monolith with npm packages pulled out by a script.